CinebaseTests/Use Cases/Movies/NowPlaying/NowPlayingPresenterSpec.swift

```diff
@@ -28,7 +28,7 @@ class NowPlayingPresenterSpec: QuickSpec {
                 mockView = MockNowPlayingView()
                 mockRouter = MockRouter()
                 mockAPIClient = MockMoviesAPIClient()
-                sut = NowPlayingPresenter(view: mockView, moviesAPIClient: mockAPIClient)
+sut = NowPlayingPresenter(view: mockView, moviesAPIClient: mockAPIClient)
```

Comment:
i inject the mockAPI to override the fetchtrailer method for the test, if not, it will init the Normal MovieAPIClient and not the Mock one.